handbook/communication/rfcs/index.md

```diff
@@ -75,15 +75,16 @@ Effective RFCs contain the following information:
 - Title that includes the RFC number.
   - The title is inlined in the Google Doc so that it is more visible and will not disappear if exported to a different format.
 - Metadata about the state of the RFC. Including but not limited to:
-  - **Editor:** The person responsible for iterating on the content of the RFC.
-    - Generally, this is the author of the Google Doc but it is possible for ownership to be transferred.
+  - **Editor:** A single person responsible for iterating on the content of the RFC.
+     - Generally, this is the author of the Google Doc but it is possible for ownership to be transferred.
+     - If there are multiple authors, they can be listed separately (e.g. under _Authors_), but a single person should be chosen and listed as the editor.
```

Comment:
What problem does this solve that the Google Docs edit history doesn’t?

Reply:
In certain cases it's useful to signal that more than one person should feel ownership over the contents. E.g., a discussion in a meeting moves into an RFC — the PM or EM might start the RFC (and be the editor) but the team really owns the content.